openbmc-env

```diff
@@ -23,8 +23,8 @@ CURR_DIR=$(pwd)
 if [ -n "$TEMPLATECONF" ]; then
     platform_name=${TEMPLATECONF//\/conf/""}
     #echo "conf:$platform_name"
-    ami=$(ls -d meta-ami/meta-*)
-    evb=$(ls -d meta-evb/meta-evb-aspeed/meta*)
+    ami=$(\ls -d meta-ami/meta-*)
+    evb=$(\ls -d meta-evb/meta-evb-aspeed/meta*)
     platform_list="${ami}"$'\n'"${evb}"
```

Comment:
Is find(1) available? what about something like `evb=$(find meta/evb-meta-evb-aspeed -type d -name 'meta*')` - I guess they are looking for directories only anyway?